### DISCUSSION TITLE
Additional SID/prompt input box, please

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#99] [Ideas]
I realized that even generating multiple images at a time(Batch size) is not enough (for me) to understand if the differences/changes between runs are good or bad for the merge.
Based on my merge experience, I'd rather have an additional input field for a SID or more than one. This will allow us to set some prefound SIDs (images) with the necessary details to compare changes.
Having an additional prompt block would allow us to keep an eye on different important details.

P.S. It seems to me that I need Z for X/Y plot :-)

@hako-mikan
@wkpark
@sverfier8807
Could you implement these to Supermerger, please?

**Comment by hako-mikan** [comment]
Adding a Z axis is time consuming, but instead you can reserve an XY plot.
And you can also specify a prompt in XY plot so you can do what you want to do.

**Comment by miasik** [comment]
I almost found a way to satisfy my need. I filled X type as "elemental":
```
cond_stage_model.transformer.text_model.encoder.layers.0:

cond_stage_model.transformer.text_model.encoder.layers.1:

cond_stage_model.transformer.text_model.encoder.layers.2:

cond_stage_model.transformer.text_model.encoder.layers.3:

cond_stage_model.transformer.text_model.encoder.layers.4:

cond_stage_model.transformer.text_model.encoder.layers.5:

cond_stage_model.transformer.text_model.encoder.layers.6:

cond_stage_model.transformer.text_model.encoder.layers.7:

cond_stage_model.transformer.text_model.encoder.layers.8:

cond_stage_model.transformer.text_model.encoder.layers.9:

cond_stage_model.transformer.text_model.encoder.layers.10:

cond_stage_model.transformer.text_model.encoder.layers.11:
```
and Y type as "seed". Working with a SID as a row is useful and convenient for me, but #104
Finally I got a grid exactly as I wanted, but it took so long!

**Comment by hako-mikan** [comment]
Do you need the feature to swap XY?

**Reply by miasik** [reply]
Sure, I do!

**Comment by miasik** [comment]
In fact, I would like to have a button for switching X/Y.
Furthermore, I have an idea.
How about the "Rotate grid" checkbox? That means X stays X and Y stays Y, but the resulting image gets rotated 90 degrees. It solves my wish to have Seeds as a row.

**Comment by hako-mikan** [comment]
Added "swap XY" option for XY plot.
